src/EditorFeatures/CSharpTest/BraceHighlighting/BraceHighlightingTests.cs

```diff
@@ -179,5 +171,12 @@ void M(int variable)
             await TestBraceHighlightingAsync("public class C [|{|]\r\n[|}|]$$");
             await TestBraceHighlightingAsync("public class C [|{|]\r\n void Foo(){}[|}|]$$");
         }
+
+        [WpfFact, Trait(Traits.Feature, Traits.Features.BraceHighlighting), Trait(Traits.Feature, Test.Utilities.Traits.Features.Tuples)]
+        public async Task TestTuples()
+        {
+            await TestBraceHighlightingAsync(@" class C { [|(|]int, int[|)|]$$ x = (1, 2); } ", TestOptions.Regular.WithTuplesFeature());
+            await TestBraceHighlightingAsync(@" class C { (int, int) x = [|(|]1, 2[|)|]$$; } ", TestOptions.Regular.WithTuplesFeature());
```

Comment:
test for nested braces? `(Foo<T>, int)`, `((int,int), int)`. I haven't exactly read the specification to know if these are legal syntax. If they are not, they can at the least be negative tests :smile: